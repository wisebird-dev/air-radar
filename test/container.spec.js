import React from 'react';
import {shallow} from 'enzyme';

import Container from '../src/components/container';

test('it works', () => {
	const wrapper = shallow(<Container/>);
	expect(wrapper).toMatchSnapshot();
});
