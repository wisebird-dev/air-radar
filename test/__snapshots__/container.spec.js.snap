// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    display: flex;
    flex-direction: column;
    width: 100%;
    margin: 0 auto 30px;
    max-width: 60em;
    padding-top: 15px;
    padding-left: 15px;
    padding-right: 15px
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
